azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/auth/AzureClientFactory.java

```diff
@@ -67,7 +51,6 @@ public static Azure getAzureClient(AuthConfiguration auth, String subscriptionId
             Log.info(String.format(SUBSCRIPTION_TEMPLATE, subscription.displayName(), subscription.subscriptionId()));
```

Comment:
move this piece of code to abstractazuremojo and record auth type, subscription id(in another PR)